packages/cli/src/cmds/preview/previewEnv.ts

```diff
@@ -437,7 +431,7 @@ export default class PreviewEnv extends YargsCommand {
       LogLevel.Warning,
       util.format(constants.manifestChangesHintMessage, `--env ${env}`)
     );
-    if (hub !== Hub.teams) {
+    if (hub !== "teams") {
```

Comment:
Would it be better to use enum such as `HubTypes.teams`? To prevent typo like `hub !== "team"`